source/codegen/metadata/nidcpower/functions_addon.py

```diff
@@ -2,4 +2,38 @@
 # Changes to the metadata should be made here, because functions.py is generated thus any changes get overwritten.
 
 functions_override_metadata = {
+    'SetRuntimeEnvironment': {
```

Comment:
Are you talking about when this metadata comes from an export (which I assume it will at some point)? I think an Issue for each one would be appropriate. Either that or something in the backlog to know that these should be removed eventually. @shastriUF , do you have a preference for how this sort of thing is tracked? It's an eventual cleanup that should be done when we get exports containing this SetRuntimeEnvironment function for each MI driver.

Reply:
Created the following issues:
https://github.com/ni/grpc-device/issues/966
https://github.com/ni/grpc-device/issues/967
https://github.com/ni/grpc-device/issues/968
https://github.com/ni/grpc-device/issues/969
https://github.com/ni/grpc-device/issues/970
https://github.com/ni/grpc-device/issues/971
https://github.com/ni/grpc-device/issues/972